scripts/main/build.js

```diff
@@ -143,17 +158,27 @@ build.album = function (data, disabled = false) {
 	return html;
 };
 
+/**
+ * @param {Photo}   data
+ * @param {boolean} disabled
+ *
+ * @returns {string} HTML for the photo
+ */
 build.photo = function (data, disabled = false) {
 	let html = "";
 	let thumbnail = "";
-	var thumb2x = "";
-	let isCover = data.id === album.json.cover_id;
-
-	let isVideo = data.type && data.type.indexOf("video") > -1;
-	let isRaw = data.type && data.type.indexOf("raw") > -1;
-	let isLivePhoto = data.livePhotoUrl !== "" && data.livePhotoUrl !== null;
-
-	if (data.sizeVariants.thumb === null) {
+	let thumb2x = "";
+	const isCover = data.id === album.json.cover_id;
+
+	const isVideo = data.type && data.type.indexOf("video") > -1;
```

Comment:
Don't we have those are return properties ? Or is it just a boolean test server side ?

Reply:
No we don't.